docs/eventing/channels/channels-crds.md

```diff
@@ -1,4 +1,10 @@
-# Available Channels
+---
+title: "Available Channels"
+#linkTitle: "Channels"
+weight: 40
+type: "docs"
+---
+
```

Comment:
It was fine as it was, this block was the old frontmatter for Hugo and isn't needed now 🙂